core/sub.mk

```diff
@@ -50,3 +50,20 @@ recipe-embedded_secure_dtb = scripts/bin_to_c.py \
 				--out $(core-embed-fdt-c)
 $(eval $(call gen-dtb-file,$(core-embed-fdt-dts),$(core-embed-fdt-dtb)))
 endif
+
+ifeq ($(CFG_SHOW_CONF_ON_BOOT),y)
+conf-mk-xz-ascii85 := $(sub-dir-out)/conf.mk.xz.ascii85
+cleanfiles += $(conf-mk-xz-ascii85)
+
+$(conf-mk-xz-ascii85): $(conf-mk-file)
+	@$(cmd-echo-silent) '  GEN     $@'
+	$(q)tail +3 $< | xz | ascii85 -w 100 >$@ || rm -rf $@
```

Comment:
What do I see? `rm -rf`? :open_mouth: Wasn't you the the side that jumped high when having `rm -rf` in scripts? :wink:   Having that said, it should be OK to use it here, since `$(conf-mk-xz-ascii85)` is defined as `conf-mk-xz-ascii85 := $(sub-dir-out)/conf.mk.xz.ascii85`meaning that if `$(sub-dir-out)` is empty, then `conf-mk-xz-ascii85` is at least set to `conf.mk.xz.ascii85` meaning that rm -rf shouldn't remove anything vital if that would happen.

Reply:
Doh! My bad. I have not changed my mind, I really meant `rm $@`, not `-rf`. 
I guess it shows that I do use `rm -rf` manually some times :wink:

Reply:
Guns are dangerous in wrong hands .. so is rm -rf :wink: !